.github/workflows/ci.yml

```
@@ -6,6 +6,24 @@ on:  # yamllint disable-line rule:truthy
   pull_request:
 
 jobs:
+  tests:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Fetch the src
+      uses: actions/checkout@v3
+    - name: Set up Python 3.11
+      uses: actions/setup-python@v4
+      with:
+        python-version: 3.11
+
+    - name: Install requirements
+      run: python -m pip install -r requirements/base.in
+           -c requirements/base.txt
```

Let's make the formatting more intuitive here:
```suggestion
      run: >-
        python -m pip install
        -r requirements/base.in
        -c requirements/base.txt
```